scripts/dice.coffee

```diff
@@ -16,42 +16,68 @@
 # Author:
 #   ab9
 #   https://github.com/github/hubot-scripts/blob/master/src/scripts/dice.coffee
+#   drobati
+#   Added fudge dice and modifiers.
+#
+# Regex Tests:
+#   https://regex101.com/r/mC7vQ3/3
 
 module.exports = (robot) ->
   robot.respond /roll (die|one)/i, (msg) ->
     msg.reply report [rollOne(6)]
   robot.respond /roll dice/i, (msg) ->
```

Comment:
A new line before all of these `robutt.respond` calls would definitely aid readability here :+1: